core/src/components/row/row.scss

```diff
@@ -2,9 +2,12 @@
 
 // Row
 // --------------------------------------------------
-
 :host {
+  --ion-grid-gap: 0px;
+
   display: flex;
 
   flex-wrap: wrap;
+
+  gap: var(--ion-grid-gap, 0px);
```

Comment:
This will always be `0px`, I believe, since you're setting this var and using it in the same block. Is this intended? Seems redundant.